### PR TITLE
[MINOR][Streaming][Docs] Minor changes on kinesis integration

### PR DESCRIPTION
**Pull request by keypointt**
## What changes were proposed in this pull request?

Some minor changes for documentation page "Spark Streaming + Kinesis Integration".

Moved "streaming-kinesis-arch.png" before the bullet list, not in between the bullets.
## How was this patch tested?

Tested manually, on my local machine.

### CODE REVIEW

**Comment by keypointt** [comment]
hi @srowen do you mind have a look at this one? very minor changes...

thank you :)

**Comment by srowen** [comment]
Seems reasonable.

**Comment by keypointt** [comment]
maybe I should also cc @tdas , he is on the streaming part

File: docs/streaming-kinesis-integration.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Here we explain how to configure Spark Streaming to receive data from Kinesis.
 
 #### Configuring Kinesis
 
-A Kinesis stream can be set up at one of the valid Kinesis endpoints with 1 or more shards per the following
+A Kinesis stream can be set up at one of the valid Kinesis endpoints with 1 or more shards following
```
Comment (srowen) [inline docs/streaming-kinesis-integration.md]: The sentence was correct before but now is ungrammatical

Comment (keypointt) [inline docs/streaming-kinesis-integration.md]: excuse me for my English...
kinesis stream can be set (...) per the guide?

Comment (srowen) [inline docs/streaming-kinesis-integration.md]: Yes, but that's only removing "following" and I don't think that's an improvement.

Comment (keypointt) [inline docs/streaming-kinesis-integration.md]: I see... I'll revert it back to original

**Comment by tdas** [comment]
LGTM. Merging to master and 2.0

**Pull request closed without merging**